src/library.js

```diff
@@ -479,7 +479,57 @@ LibraryManager.library = {
 #endif
   },
 
-  emscripten_resize_heap__deps: ['emscripten_get_heap_size', '$abortOnCannotGrowMemory'],
+#if TEST_MEMORY_GROWTH_FAILS
+  _reallocBuffer: function(size) {
+    return null;
+  },
+#else
+
+  _reallocBuffer: function(size) {
```

Comment:
how about calling this `$emscripten_realloc_buffer`?

Reply:
Ok.